pr2eus_tutorials/euslisp/pr2-tabletop-object-grasp.l

```diff
@@ -210,15 +210,15 @@
   (setq *grasp-state* "returning")
   (publish-grasp-state)
   (pr2-pregrasp-pose)
-  (publish-info "click tabletop object in Rviz!")
-  (publish-info "click tabletop object in image_view2!")
+  (ros::ros-info "click tabletop object in Rviz or in image_view2!")
+  (publish-status 'info (concatenate string "click tabletop object" '(#\newline) "in Rviz or in image_view2!"))
```

Comment:
may be `(format nil "click tabletop object~%in Rviz or in image_view2!")` seems simpler

Reply:
Thank you for your advice, I improve it.